src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnGateway.md

```diff
@@ -0,0 +1,279 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/new-azurermvpngateway
+schema: 2.0.0
+---
+
+# New-AzureRmVpnGateway
+
+## SYNOPSIS
+Creates a Scalable VPN Gateway.
+
+## SYNTAX
+
+### ByVirtualHubName (Default)
+```
+New-AzureRmVpnGateway -Name <String> -ResourceGroupName <String> -VpnGatewayScaleUnit <UInt32>
```

Comment:
Looks like the order on this is wrong.  Please always have "ResourceGroupName" then "<parent>Name" then "Name", then any other parameters.  This depends on the order in your cs file so please update that and then regenerate help.  Apply this for everything - identity parameters should always be first, ordered by top level to bottom level, then all other parameter follow.

Reply:
Regenerated help. Was fixed in code.